recipe/meta.yaml

```
@@ -21,9 +22,11 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - setuptools
+    - packaging
```

```suggestion
    - packaging
    - importlib-metadata. # [py<38]
  run_constrained:
    - gevent >=1.4.0
    - eventlet >=0.24.1
    - tornado >=0.2
```
https://github.com/benoitc/gunicorn/blob/21.2.0/setup.py#L73-L84